package.json

```diff
@@ -7,6 +7,7 @@
     "bootstrap": "^3.3.7",
     "classnames": "^2.2.5",
     "flux": "^3.1.2",
+    "history": "^4.6.1",
```

Comment:
This is required since I am not using react-router, to instantiate the history manually. This is also suggested in the package that I am using.